### PR TITLE
Fix issues in EIB quickstart

### PR DESCRIPTION
**Pull request by atanasdinov**
- First of likely many PRs which addresses most of the API changes and TODOs in the initial version

### CODE REVIEW

File: asciidoc/quickstart/eib.adoc [hunk 1]
```diff
@@ -136,78 +137,70 @@ operatingSystem:
...
 To build the image, we can run:
 
 [,bash]
 ----
-docker run --rm -it -v $PWD/eib/:/eib \
- registry.opensuse.org/home/atgracey/eib/container/containerfile/eib:latest \
- /usr/bin/eib -config-file eib-config.yaml -config-dir /eib -build-dir \
- /eib/_build
+docker run --rm --privileged -it -v $PWD/eib/:/eib \
+ registry.opensuse.org/isv/suse/edge/edgeimagebuilder/containerfile/suse/edge-image-builder:1.0.0 \
```
Comment (atanasdinov) [inline asciidoc/quickstart/eib.adoc:199]: This will eventually work and I don't mind merging it like this.

File: asciidoc/quickstart/eib.adoc [hunk 2]
```diff
@@ -13,11 +13,11 @@ The Edge Image Builder is a tool that allows for generating a fully customized d
 
 It supports a variety of configuration options including (but not limited to):
 
-* Users, passwords, and ssh-keys
+* Groups, users, passwords, ssh-keys
```
Comment (agracey) [inline asciidoc/quickstart/eib.adoc:16]: Nitpick: I would probably switch the order to `Users, groups, ...`. It reads more fluidly.

File: asciidoc/quickstart/eib.adoc (hunk 1 above)
Comment (agracey) [reply asciidoc/quickstart/eib.adoc:199]: Yay! I'm just glad it's moving away from my project 😌

**Review by agracey — APPROVED** [review]
Good clean up

**Pull request merged**